pennylane/ops/qubit.py

```diff
@@ -1882,6 +1882,39 @@ def diagonalizing_gates(self):
         return [QubitUnitary(self.eigendecomposition["eigvec"].conj().T, wires=list(self.wires))]
 
 
+# =============================================================================
+# Measurements
+# =============================================================================
+
+
+class Measure(Projection):
+    r"""Measure(wires)
+    A measurement in the computational basis.
+
+    **Details:**
+
+    * Number of wires: Any
+    * Number of parameters: 0
+    * Gradient recipe: None
+
+    Args:
+        wires (Sequence[int] or int): the wire(s) the operation acts on
```

Comment:
Perhaps we could add an example usage below here? It'll help users understand that this is for measurements partway through the circuit